documentation/endpoints/signups.md

```diff
@@ -24,7 +24,7 @@ Example Response:
     },
     {
       "id": 2,
-      "northstar_id": "5589c991a59dbfa93d8b45ae",
+      "northstar_id": "345367",
```

Comment:
Let's leave these as Mongo IDs to make it clearer what a real response would look like!

Reply:
ok! I wasn't sure if that was dangerous or not but will change back!